src/mg/PAMI/Client/Impl/ClientImpl.php

```diff
@@ -185,9 +189,7 @@ public function open()
 	    @stream_set_blocking($this->_socket, 0);
 	    $this->_currentProcessingMessage = '';
 	    //register_tick_function(array($this, 'process'));
-	    if ($this->_logger->isDebugEnabled()) {
```

Comment:
Well, that's not what a standard PSR 3 logger does either :). It's a very basic interface. But I guess we can change the message building to `"Note {$variable} something"` syntax. Is that what you mean?